seaborn/categorical.py

```diff
@@ -2525,12 +2575,12 @@ def countplot(x=None, y=None, hue=None, data=None, order=None, hue_order=None,
     elif x is not None and y is not None:
         raise TypeError("Cannot pass values for both `x` and `y`")
     else:
-        raise TypeError("Must pass valus for either `x` or `y`")
+        raise TypeError("Must pass values for either `x` or `y`")
 
     plotter = _BarPlotter(x, y, hue, data, order, hue_order,
                           estimator, ci, n_boot, units,
                           orient, color, palette, saturation,
-                          errcolor)
+                          errcolor, None, None)
```

Comment:
Hopefully this is sufficient for the tests to pass... If not, please let me know how this should be done